docs/business-finder-link-based-facets.md

```diff
@@ -0,0 +1,51 @@
+
+This view only applicable to the business readiness finder, but is not yet a candidate for deletion, as it's been proven to be a useful feature.
+
+### From the content designer perspective
+
+In most finders, facets are published as part of the content item itself. This means when the content item is updated via its publishing application, the facets are updated too and associated with the change note accompanying the change. This triggers an email to subscription lists which are subscribed to that facet, and the change note appears in the email body.
+
+The business finder retrieves content from multiple different publishing apps (via `search-api`), therefore facets are published independently:
+
+- There is a ['facet group' associated with the business finder](https://content-tagger.integration.publishing.service.gov.uk/facet_groups). The facet group has a number of possible facets (e.g. "Sector / Business Area"), each of which have a number of possible facet values (e.g. "Accommodation"), which is a [manually curated list in YAML](https://github.com/alphagov/content-tagger/blob/master/lib/data/find-eu-exit-guidance-business.yml).
+- Content designers follow the [How to tag content with facet values](https://docs.publishing.service.gov.uk/manual/tag-content-with-facet-values.html) instructions to tag a particular piece of content (e.g. "/government/publications/workplace-rights-if-theres-no-brexit-deal") with a particular facet value (e.g. "Accommodation").
+- Content designers can choose whether or not to trigger an email alert for the tagging by checking the `Notify email subscribers that this content has been tagged` box and filling in a change note.
+
+The content will now appear in the search results of the business finder when that facet value is selected.
+
+## How this works technically
+
+Standards finders use facets that have been published as part of the content item itself, which therefore appear under `details["facets"]` in the [content item JSON](https://www.gov.uk/api/content/cma-cases).
+
+The business finder uses facets that reference 'links' published by content-tagger, which therefore appear under `links["facet_group"]` in the [business finder content item JSON](https://www.gov.uk/api/content/find-eu-exit-guidance-business).
+
+Let's call these "details-based facets" and "link-based facets" respectively.
+
+Details-based facets are part of the content item itself, so when the content item is updated via its publishing API, they're updated too and associated with the change note accompanying the change. This triggers an email to subscription lists which are subscribed to that facet, and the change note appears in the email body.
+
+This wouldn't work very well for the business finder, as we would have had to add an interface to every publishing app.
+
+The link-based facets approach means:
+
+1. We don't have to change any publishing app interface.
```

Comment:
Should be "all" and interfaces should be plural.

Reply:
Have pluralised 👍 think 'any' is better than 'all', as the latter suggests we do have to change _some_ publishing app interfaces (and AFAIK, we don't)

Reply:
"Any" is incorrect though, because content-tagger is a publishing-app and it's been changed.